fastly/block_fastly_service_v1_waf.go

```diff
@@ -44,6 +44,12 @@ func (h *WAFServiceAttributeHandler) Register(s *schema.Resource) error {
 					Computed:    true,
 					Description: "The Web Application Firewall (WAF) ID",
 				},
+				"disabled": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "A flag used to completely disable a Web Application Firewall. This is intended to be used as an emergency.",
```

Comment:
I know not originally your message, but we may as well clean it up. I think this reads better.

```suggestion
					Description: "A flag used to completely disable a Web Application Firewall. This is intended to only be used in an emergency.",
```